### DISCUSSION TITLE
Plugin Settings Presets/Profile tweak

### DISCUSSION THREAD

**Discussion by Bigbaddrag** [runelite/runelite#17075] [Ideas]
I have a few different plugin settings combos I use, where most things I want to stay the same, only changing a few settings. For example, I have one that has a bigger client size with more infoboxes enabled, and one that is smaller with some infoboxes disabled to clear up screenspace. If I were to use profiles, those would save every plugin setting as a whole, so if I want to make a change to any other plugin, I would need to change it once, switch to the other profile, and change that setting again. If I add a third profile for something else, like enabling some snowflake iron plugins, then that's another profile I would need to update.

If there was a feature similar to profiles (which I'll call presets here), but instead of saving every setting as a whole you choose specific plugins from what you have installed, and the settings for those plugins would be saved as a preset. With a saved preset, it would have something like an Apply button, which would update only the chosen plugins to use the saved settings, while leaving other plugins unchanged.

Using the three profile examples above, something like this would allow me to choose the smaller client size and also the snowflake plugins, and switch back to larger client size without interfering with the snowflake plugins.

**Comment by antero111** [comment]
Hi, see https://runelite.net/plugin-hub/show/plugin-presets on the plugin hub. It's a preset plugin that predates profiles in a way that you can store certain settings to presets and change between them, not all at once like profiles does. With Plugin Presets, you can make a presets that you can toggle between for just some certain settings e.g. client size like you wanted.

For account specific presets, loot tracker, timers etc. use profiles.

**Reply by Bigbaddrag** [reply]
That's exactly what I was thinking of, I can't believe I never saw that one, thanks!
